neard/src/cli.rs

```diff
@@ -454,6 +463,21 @@ fn init_logging(verbose: Option<&str>) {
         .init();
 }
 
+#[derive(Clap)]
+pub(super) struct RecompressStorageSubCommand {
```

Comment:
I'm wondering if we should it put it inside nearcore? or have a separate tool to do only this.

Reply:
-1 on a separate tool, I feel it’s overall better to have a single Swiss Army knife binary:

* (much) faster linking times
* easier incident investigation, no need to download extra tools
* better project structure: it’s easier to notice a dead module in an actively worked upon crate than to note a dead crate.